e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -270,10 +276,49 @@ describe("scenarios > embedding > full app", () => {
     });
 
     it("should hide the dashboard header by a param", () => {
-      visitDashboardUrl({ url: "/dashboard/1", qs: { header: false } });
+      visitDashboardUrl({
```

Comment:
The test is pretty much the same as the one in https://github.com/metabase/metabase/pull/40166